src/components/Lottie/index.tsx

```diff
@@ -27,13 +31,33 @@ function Lottie({source, webStyle, ...props}: Props, ref: ForwardedRef<LottieVie
         setAnimationFile(source.file);
     }, [setAnimationFile, source.file]);
 
+    useEffect(() => {
+        const unsubscribeNavigationFocus = navigation.addListener('focus', () => {
+            setIsHidden(false);
+        });
+        return unsubscribeNavigationFocus;
+    }, [navigation]);
+
+    // Prevent the animation from running in the background after navigating to other pages.
+    // See https://github.com/Expensify/App/issues/47273
+    useEffect(() => {
+        const unsubscribeNavigationBlur = navigation.addListener('blur', () => {
+            const state = navigation.getState();
+            const targetRouteName = state?.routes?.[state?.index ?? 0]?.name;
+            if (targetRouteName !== NAVIGATORS.RIGHT_MODAL_NAVIGATOR) {
+                setIsHidden(true);
```

Comment:
This change causes an issue where assets are hidden during the screen transition, and in some cases, if the transition is slow, users may briefly see a blank view.​